### ISSUE TITLE
Pyright doesn't recognize the `__members__` attribute of enumeration class

### ISSUE THREAD

**Issue by MapleCCC** [microsoft/pyright#759]
**Describe the bug**
According to Python official document (https://docs.python.org/3.8/library/enum.html#iteration), there is an attribute `__members__` of every enumeration class that provides all names defined in the enumeration class. Since it appears in the official document, we should consider it a public API. But Pyright doesn't seem to recognize the `__members__` attribute, as elaborated in the following screenshot.

**To Reproduce**
```python
from enum import Enum

class Fruit(Enum):
    apple = 1
    orange = 2
    pear = 3

print(Fruit.__members__)
```

**Expected behavior**
Pyright should be able to recoginize `__members__` as a mapping.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/25131775/85567329-5a3a2900-b663-11ea-9bcb-e066dd8f2e1a.png)
![image](https://user-images.githubusercontent.com/25131775/85567542-89509a80-b663-11ea-9a1b-69791237d783.png)

**VS Code extension or command-line**
Pyright is run as VS Code extension.
Extension version 1.1.44

**Comment by jakebailey** [comment]
Typeshed's definition of enum includes it in the metaclass: https://github.com/python/typeshed/blob/master/stdlib/3/enum.pyi

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in Pyright version 1.1.48, which I just published. It will also be included in the next public release of Pylance.
